src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/NewLineUserSettingFormattingRule.cs

```diff
@@ -511,6 +558,7 @@ public bool Equals(CachedOptions other)
                     && NewLinesForBracesInTypes == other.NewLinesForBracesInTypes
                     && NewLinesForBracesInAnonymousTypes == other.NewLinesForBracesInAnonymousTypes
                     && NewLinesForBracesInObjectCollectionArrayInitializers == other.NewLinesForBracesInObjectCollectionArrayInitializers
+                    && NewLinesForBracesInWithInitializer == other.NewLinesForBracesInWithInitializer
```

Comment:
Looks like this is one of the places where a record is suitable to avoid having to update GetHashCode and Equals implementation every time a field is added.